atlasdb-ete-tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:x86_64-debian-jre-11.0.18_10
+FROM azul/zulu-openjdk:17-latest
```

Comment:
Maybe some fixed version?

Reply:
Discussions with internal Java distribution experts suggest this is only used to bootstrap Gradle, so fine.